### PR TITLE
Add test utils and organization crud resolver tests

### PR DESCRIPTION
**Pull request by golanglemonade**
Adds unit tests for the organization resolvers

### CODE REVIEW

File: internal/api/test.db
Comment (matoszz) [inline internal/api/test.db:1]: should this be pushed or added to `.gitignore`?

File: internal/api/tools_test.go [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+	ent "github.com/datumforge/datum/internal/ent/generated"
+	"github.com/datumforge/datum/internal/entdb"
+)
+
+var (
+	defaultDBURI = "file:ent?mode=memory&cache=shared&_fk=1"
+	EntClient    *ent.Client
+)
+
+func TestMain(m *testing.M) {
```
Comment (matoszz) [inline internal/api/tools_test.go:29]: does this get run with the tasks that are already in our taskfile?

File: internal/api/test.db
Comment (golanglemonade) [reply internal/api/test.db:1]: whoops, no that was from testing, I'll remove.

Comment (golanglemonade) [reply internal/api/test.db:1]: yeah, it will run with `task go:test`

File: internal/api/tools_test.go (hunk 1 above)
Comment (golanglemonade) [reply internal/api/tools_test.go:29]: yeah it will run with `task go:test`

**Review by matoszz — APPROVED** [review]

**Pull request merged**